config/api.yml

```diff
@@ -200,6 +200,29 @@
     :options:
     - :primary
     :verbs: *gd
+  :authentications:
+    :description: Authentications
+    :options:
+    - :collection
+    :verbs: *gpd
+    :klass: Authentication
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: embedded_automation_manager_credentials_view
```

Comment:
The thought behind #13827 is that higher level collections are authorizable if the user has at least one of the related/relevant roles, so if a user has embedded_automation_manager_credentials_view or later automation_manager_credentials_view, etc. he'd be able to access this collection.

Signature above is fine, once #13827 is merged, you'd also use the array equivalent:

```
  :get:
  - :name: read
     :identifier:
     - embedded_automation_manager_credentials_view
     - automation_manager_credentials_view  (adding later whatever additional ones materializes as )
```